mods/common/chrome/lobby-servers.yaml

```diff
@@ -106,17 +106,17 @@ Container@LOBBY_SERVERS_BIN:
 							TooltipTemplate: SIMPLE_TOOLTIP
 							TooltipText: Requires OpenRA forum account
 						LabelWithTooltip@PLAYERS:
-							X: 290
+							X: 290+92
```

Comment:
Representative question for all these +/- cases:
Didn't we at some point introduce a chrome lint rule or something that expects spaces around those?

See `common/chrome/lobby.yaml` line 3 for example:
`X: (WINDOW_RIGHT - WIDTH) / 2`

Reply:
Whoops, I was supposed to have squashed all of these calculations before PRing. Looks like I missed these header rows.